src/freenet/node/simulator/RealNodeRequestInsertTest.java

```diff
@@ -172,8 +176,8 @@ int insertRequestTest() throws CHKEncodeException, InvalidCompressionCodecExcept
         Node randomNode = nodes[node1];
         //Logger.error(RealNodeRequestInsertTest.class,"Inserting: \""+dataString+"\" to "+node1);
         
-        //boolean isSSK = requestNumber % 2 == 1;
-        boolean isSSK = true;
+        boolean isSSK = requestNumber % 2 == 1;
+        //boolean isSSK = true;
```

Comment:
Why keep both of these? The one switched to seems like a more general test - is that always preferable?